src/pytorch_lightning/loggers/wandb.py

```diff
@@ -26,10 +27,26 @@
 from pytorch_lightning.callbacks.model_checkpoint import ModelCheckpoint
 from pytorch_lightning.loggers.logger import Logger, rank_zero_experiment
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
-from pytorch_lightning.utilities.imports import _WANDB_GREATER_EQUAL_0_10_22, _WANDB_GREATER_EQUAL_0_12_10
+from pytorch_lightning.utilities.imports import _RequirementAvailable
 from pytorch_lightning.utilities.logger import _add_prefix, _convert_params, _flatten_dict, _sanitize_callable_params
 from pytorch_lightning.utilities.rank_zero import rank_zero_only, rank_zero_warn
 
+
+@lru_cache
+def _is_wandb_available():
+    return _RequirementAvailable("wandb")
+
+
+@lru_cache
+def _is_wandb_greater_equal_0_10_22():
+    return _RequirementAvailable("wandb>=0.10.22")
+
+
+@lru_cache
+def _is_wandb_greater_equal_0_12_10():
+    return _RequirementAvailable("wandb>=0.12.10")
```

Comment:
```suggestion
_WANDB_GREATER_EQUAL_0_12_10 = _RequirementAvailable("wandb>=0.12.10")
```

And then in the code below you don't need to make any changes